geode-core/src/main/java/org/apache/geode/cache/query/internal/cq/CqServiceProvider.java

```diff
@@ -59,7 +60,7 @@ public static CqService create(InternalCache cache) {
       return new MissingCqService();
     }
 
-    return factory.create(cache);
+    return factory.create(cache, CommandInitializer.getDefaultInstance());
```

Comment:
is there a way to do this without calling the deprecated `getDefaultInstance`?

Reply:
Not without a much deeper refactoring. We will need a way to connect two very distant components and that is just out of scope for what I want to get done now. So it's no worse than it was. It should discourage continued abuse. It is now testable. So I see a net positive here.

Reply:
okey dokey